### PR TITLE
Fixed and rearranged unit tests

### PR DESCRIPTION
**Pull request by bjcmit**
- Fixed test_train_model test in code_test.py
- Renamed code_test.py to train_test.py
- Moved train_test.py under diabetes_regression directory
- Moved data_test.py under data directory
- Added lint and unit test tasks to azdo-base-pipeline.yml
- Deleted lint-and-test.sh and tox.ini

### CODE REVIEW

File: .pipelines/azdo-base-pipeline.yml [hunk 1]
```diff
@@ -1,7 +1,17 @@
 steps:
-- script: |
-    ./lint-and-test.sh
-  displayName: 'Linting & unit tests'
+- task: PythonScript@0
+  inputs:
+    scriptSource: inline
+    script: set -eux
+            flake8 --output-file=lint-testresults.xml --format junit-xml
```
Comment (dtzar) [inline .pipelines/azdo-base-pipeline.yml]: [This fails](https://aidemos.visualstudio.com/MLOps/_build/results?buildId=3772&view=logs&j=12f1170f-54f2-53f3-20dd-22fc7dff55f9&t=0d5f46f2-efd2-5fe7-3c1a-f6adc81b525b). Perhaps use the following which adds the | :
```
- script: |
   flake8 --output-file=$(Build.BinariesDirectory)/lint-testresults.xml --format junit-xml  
```

File: .pipelines/azdo-base-pipeline.yml [hunk 2]
```diff
@@ -1,7 +1,17 @@
...
+  displayName: 'Run lint tests'
+
+- task: PythonScript@0
+  inputs:
+    scriptSource: inline
+    script: set -eux
```
Comment (dtzar) [inline .pipelines/azdo-base-pipeline.yml]: Haven't tested this yet, but my guess is you'll need to remove `set -eux` and add a `|` or move to a single line

**Comment by sudivate** [comment]
@bjcmit  now that you have moved the test files to the respective project directory. Would you like to consider deleting the folder at the root?

**Comment by dtzar** [comment]
[Build ](https://aidemos.visualstudio.com/MLOps/_build/results?buildId=3780&view=ms.vss-test-web.build-test-results-tab&runId=1002070&resultId=100008&paneView=debug) fails lint `./diabetes_regression/training/test_train.py:27:73: W292 no newline at end of file`

**Comment by dtzar** [comment]
> @bjcmit now that you have moved the test files to the respective project directory. Would you like to consider deleting the folder at the root?

Yes, we should remove that root test/unit directory.  I believe if you delete init.py there it should go away.

**Pull request merged**